src/style.css

```diff
@@ -1,3 +1,80 @@
+/* Moyo */
+
+html {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+ul {
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: white;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+  text-transform: uppercase;
+  margin: 0;
+}
+
+.nav__item {
+  position: relative;
+}
+
+.nav__link {
+  display: inline-block;
+  position: relative;
+  text-decoration: none;
+  font-size: 12px;
+  color: black;
+  height: 60px;
+  line-height: 60px;
+  margin: 0 10px;
```

Comment:
add margin to items instead of links to avoid margin collapsing. 
Also you can use `:not(:last-child)` to set margin to all items besides last one
```suggestion
```